### PR TITLE
feat/AUT-1612/allow set QTI Identifier pattern from configuration

### PR DESCRIPTION
**Pull request by bziondik**
Related to: https://oat-sa.atlassian.net/browse/AUT-1612
Depends on: 

- [x] https://github.com/oat-sa/extension-tao-itemqti/pull/2007
- [x] update in composer version of taoQtiItem

Added support to update QTI Identifier pattern from configuration for Item and Test Authoring

**How to test:**
1. set in config new pattern `config/tao/client_lib_config_registry.conf.php`
```
        'taoQtiItem/qtiCreator/widgets/helpers/qtiIdentifier' => array(
            'qtiIdPattern' => '^[a-zA-Z_][a-zA-Z0-9_-]*$'
        )
```
2. check that in Item and Test Authoring QTI identifier for test, test-part, test-section and assessmentItemRef with dot is not valid (error message will stay the same)

### CODE REVIEW

**Comment by pnal** [comment]
Do we need to update composer dependency version of oat-sa/extension-tao-itemqti?

**Review by Silvia-loza — APPROVED** [review]
Works as expected in local for both envs Udir and tao-community
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [X] New code is respecting code style rules
- [X] New code is respecting best practices
- [X]  Pull request's target is not ` master`
- [X] Feature is working correctly on my local machine (if applicable)
- [X] Acceptance criteria are respected
- [X] Pull request title and description are meaningful

**Comment by bziondik** [comment]
> 

Thanks! yes, will do after releasing taoQtiItem

**Review by clagosarias — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [ ]  New code is not subject to concurrency issues (if applicable)
- [ ]  The feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request's target is not master

File: views/js/controller/creator/helpers/validators.js [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+                        rubricBlock,
+                        'rubricBlock.div.feedbackBlock',
+                        'content'
+                    );
+                    if (feedbackBlock && !outcomes[feedbackBlock.outcomeIdentifier]) {
+                        throw new Error(
+                            __(
+                                'The outcome "%s" does not exist, but it is referenced by a feedback block!',
+                                feedbackBlock.outcomeIdentifier
+                            )
```
Comment (augustas) [inline views/js/controller/creator/helpers/validators.js:149]: Translatable string parser can't identify multiline strings, please, if you have the possibility, make this expression in 1 line, like it was before refactoring. Thx!

**Pull request merged**